src/common/lang/en.js

```diff
@@ -297,13 +297,13 @@ module.exports = {
     newAddressLbl: 'New Address',
     activeAddressLbl: 'Active Address',
     blockLbl: 'New Block',
-    newOntIdLbl: 'New ONT-ID',
-    activeOntIdLbl: 'Active ONT-ID',
+    newOntIdLbl: 'New ONYX-ID',
+    activeOntIdLbl: 'Active ONYX-ID',
```

Comment:
In other lines "ONYX ID" without hyphen

Reply:
I followed original style of names, but hyphen can be removed.